interp/src/lib.rs

```diff
@@ -8,3 +8,6 @@ pub mod serialization;
 mod tests;
 
 pub mod flatten;
+
+// ethan: griffin ok'd this
```

Comment:
Remove this comment please. (Next time, just add a comment on the PR instead of the source code).